builtin/providers/datadog/resource_datadog_timeboard.go

```diff
@@ -443,46 +441,46 @@ func buildGraphs(terraformGraphs *[]interface{}) *[]datadog.Graph {
 				PaletteFlip *bool   `json:"paletteFlip,omitempty"`
 			}{}
 
-			if palette_, ok := s["palette"]; ok {
-				palette := palette_.(string)
+			if v, ok := s["palette"]; ok {
+				palette := v.(string)
 				style.Palette = &palette
```

Comment:
When the Graph struct has been broken up in https://github.com/zorkian/go-datadog-api/issues/98 we'll have generated Setters that do this tedious and error prone work for us.